novawallet/Modules/DelegatedAccounts/Multisig/MultisigOperationFetchProxy/MultisigOperationFetchProxyWireframe.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+final class MultisigOperationFetchProxyWireframe {
+    let flowState: MultisigOperationsFlowState?
+
+    init(flowState: MultisigOperationsFlowState?) {
+        self.flowState = flowState
+    }
+}
+
+// MARK: - MultisigOperationFetchProxyWireframeProtocol
+
+extension MultisigOperationFetchProxyWireframe: MultisigOperationFetchProxyWireframeProtocol {
+    func close(from view: ControllerBackedProtocol?) {
+        view?.controller.dismiss(animated: true)
+    }
+
+    func showConfirmationData(
+        from view: ControllerBackedProtocol?,
+        for operation: Multisig.PendingOperationProxyModel,
+        flowState: MultisigOperationsFlowState
+    ) {
+        guard let confirmView = MultisigOperationConfirmViewFactory.createView(
+            for: operation,
+            flowState: flowState
```

Comment:
It looks weird to make a trip interactor -> presenter -> wireframe for the flow state. I would rather create the state directly in the view factory, store `MultisigOperationProviderProxy` in it and pass state further. Then we will have shared provider for both operations and operation details. WDYT?